Script Analysis Examples/Test Data Analysis/Failure_Pareto_Result_Analysis.ipynb

```diff
@@ -0,0 +1,529 @@
+    "from requests.packages.urllib3.exceptions import InsecureRequestWarning\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import systemlink.clients.nitestmonitor as testmon\n",
+    "import systemlink.clients.nifile as nifile\n",
+    "\n",
+    "requests.packages.urllib3.disable_warnings(InsecureRequestWarning)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parameters\n",
+    "- `result_ids`: IDs of the test results.\n",
+    "\n",
+    "Parameters are also listed in the metadata for the parameters cell, along with their default values. The Notebook Execution services uses that metadata to pass parameters from the Test Monitor Reports page to this notebook. To see the metadata, select the code cell and click the wrench icon in the far left panel.\n",
+    "\n",
+    "Sample metadata:\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "  \"papermill\": {\n",
+    "    \"parameters\": {\n",
+    "      \"result_ids\": []\n",
+    "    }\n",
+    "  },\n",
+    "  \"systemlink\": {\n",
+    "    \"namespaces\": [],\n",
+    "    \"parameters\": [\n",
+    "      {\n",
+    "        \"display_name\": \"result_ids\",\n",
+    "        \"id\": \"result_ids\",\n",
+    "        \"type\": \"string[]\"\n",
+    "      }\n",
+    "    ],\n",
+    "    \"version\": 2\n",
+    "  },\n",
+    "  \"tags\": [\"parameters\"]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "For more information on how parameterization works, review the [papermill documentation](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#how-parameters-work)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "result_ids": []
+     }
+    },
+    "systemlink": {
+     "namespaces": [
+      "ni-testmanagement"
+     ],
+     "outputs": [],
+     "parameters": [
+      {
+       "display_name": "result_ids",
+       "id": "result_ids",
+       "type": "string[]"
+      }
+     ],
+     "version": 2
+    },
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "result_ids = [\"aaf89284-d1c6-4276-ae03-aa3fcf5d1933\",\"6b108caf-877e-4992-92ba-743fd3e49671\",\"ad22b2f2-bb2c-4bd0-8a88-080b6df4a645\",\"963f076d-4a2f-4740-874d-1aadf6fb451a\",\"6225324c-c37e-44f5-9780-a469795bf5a7\",\"070e706c-8041-474d-bfba-f869717431a0\",\"8774433f-5b77-480b-86b1-cebe60e84852\",\"b595ace2-422c-4693-8e17-d50f79e1784d\",\"90834263-ddc7-4c33-aa30-62f353ae17a3\",\"cd4c65f6-d402-4642-9957-c6515fb4946c\",\"a50793a1-d844-4ade-8f9c-69befa5fad39\",\"f1a8a2e8-289d-42e0-8305-447248db93f7\",\"c6732428-55b8-4dfd-bedf-dc0e646eaf34\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_key = os.getenv(\"SYSTEMLINK_API_KEY\")\n",
```

Comment:
I don't think any of these constants are actually used in the rest of the example. Can you please check and make sure that we remove any dead code like this? 

It's important that these examples are easy to understand and extra code like this makes it confusing to the user.

Reply:
Removed the dead code from the Notebooks